test/Directory.Build.targets

```diff
@@ -2,6 +2,11 @@
 
   <Import Project="..\Directory.Build.targets" />
 
+  <PropertyGroup>
+    <!-- This is required in order to have following extension files in the output folder of the projects referencing other projects. -->
```

Comment:
💡 Would be good to have a comment explaining that this needs to be in **Directory.Build.targets** or the default value for AllowedReferenceRelatedFileExtensions will not be applied.